src/styles/responsive.scss

```diff
@@ -0,0 +1 @@
+$mobile-version: "only screen and (max-width: calc(600 / 16 * 1rem))";
```

Comment:
question: Can you explain what this calc does and what's its benefit compared to normal `37.5rem`? And why rem instead of px? :)

Reply:
To answer your first question: calc allows us to do maths, here it allows us to convert px to rem.
Then, to answer you second question about why I used rem instead of px : 

with a 16px base font size, 68.75rem equals 1100px (68.75 * 16). If our window is 1200px, we'll get the desktop styles.
But, if the user cranks their default font size to 24px, the math changes. Now, this media query only triggers at 1650px (68.75 * 24). At 1200px, we're getting the tablet styles, not the desktop ones.
In other words, rem-based viewports scale with the user's chosen default font size. As the font size gets bigger, the breakpoints slide up the scale. (From Josh Commeau course)